android/javatests/org/chromium/chrome/browser/BytecodeTest.java

```diff
@@ -2034,6 +2047,10 @@ public void testFieldsExist() throws Exception {
                 fieldExists(
                         "org/chromium/chrome/browser/ui/system/StatusBarColorController",
                         "mBackgroundColorForNtp"));
+        Assert.assertTrue(
+                fieldExists(
+                        "org/chromium/chrome/browser/tasks/tab_management/IncognitoTabSwitcherPane",
+                        "mReferenceButtonData"));
```

Comment:
Since we set it via refllection, we should also check that its type hasn't changed. Otherwise we will get runtime errors which are quite hard to catch.

Reply:
thanks, updated test at 
https://github.com/brave/brave-core/pull/25291/commits/50e606217353b5915de8737b3cdf018e9ebe7ae1